packages/react/test/setup.ts

```diff
@@ -17,3 +17,15 @@ jest.mock('react', () => ({
     ...jest.requireActual('react'),
     useLayoutEffect: jest.requireActual('react').useEffect,
 }));
+
+{
+    let counter = 0;
+    jest.mock('../src/utils/uuid', () => ({
+        // eslint-disable-next-line no-plusplus
```

Comment:
Oh, really?

Reply:
Yep I agree this rule must be disabled. 
I never saw anyone making the mistake as described: https://eslint.org/docs/latest/rules/no-plusplus.

Reply:
Yeah, that rule should have been `no-space-around-plusplus` 😄